src/hotspot/share/jeandle/jeandleRuntimeRoutine.hpp

```diff
@@ -46,9 +46,17 @@
   def(exceptional_return)
```

Comment:
We also need define assembly routines with argument type and return type. For example, StubRoutines::_dsin has signature of `double dsin(double)`

Reply:
Sure we need this, but I think maybe it's better to support it when we need it. If we support it now, we don't have use case and test case.
What do you think?

Reply:
I just find Jeandle use stub routines defined by JeandleRuntimeRoutine. How about routines defined by hotspot? Could we use them directly? Or do we need re-implement them in jeandle ? 
I prefer to reuse them. So we may need some class like `CallHotspotVM` or `CallHotspotRoutine`?

Reply:
We can call them directly, just use a routine call like call a JeandleRuntimeRoutine.